core-services/image-mirroring/openshift/mapping_origin_4_8

```diff
@@ -137,15 +111,8 @@ registry.ci.openshift.org/origin/4.8:hyperconverged-cluster-operator-tests quay.
 registry.ci.openshift.org/origin/4.8:hyperconverged-cluster-webhook quay.io/openshift/origin-hyperconverged-cluster-webhook:4.8 quay.io/openshift/origin-hyperconverged-cluster-webhook:4.8.0
 registry.ci.openshift.org/origin/4.8:hyperkube quay.io/openshift/origin-hyperkube:4.8 quay.io/openshift/origin-hyperkube:4.8.0
 registry.ci.openshift.org/origin/4.8:insights-operator quay.io/openshift/origin-insights-operator:4.8 quay.io/openshift/origin-insights-operator:4.8.0
-registry.ci.openshift.org/origin/4.8:installer quay.io/openshift/origin-installer:4.8 quay.io/openshift/origin-installer:4.8.0
```

Comment:
This seems... weird.  Pretty sure we still have an installer.

Reply:
If this is OKD, then this may be because of the same things for machine-config-operator etc.

Reply:
It seems so. RC excludes it

https://github.com/openshift/release/blob/8e016f02862fdd5831d8d7b0a0c2aca09bde3243/core-services/release-controller/_releases/release-ocp-4.8-ci.json#L15-L19

And @vrutkovs builds it locally?


```oc get is -n origin 4.8 -o yaml | yq -y '.status.tags[]|select(.tag=="installer")'
items:
  - created: '2021-08-04T09:25:58Z'
    dockerImageReference: quay.io/vrutkovs/okd-installer@sha256:0195d0f538f0e6b94c723600d9bcfc51d219e5fd1559acb0b55408b666768e7d
    generation: 2354
    image: sha256:0195d0f538f0e6b94c723600d9bcfc51d219e5fd1559acb0b55408b666768e7d
  - created: '2021-08-04T09:14:59Z'
    dockerImageReference: quay.io/vrutkovs/okd-installer@sha256:29833e08594b2efc8c3f2a4b408b3950d3ede77d6eb1909c6b405755ad804353
    generation: 2350
    image: sha256:29833e08594b2efc8c3f2a4b408b3950d3ede77d6eb1909c6b405755ad804353
  - created: '2021-08-01T11:27:47Z'
    dockerImageReference: quay.io/vrutkovs/okd-installer@sha256:0195d0f538f0e6b94c723600d9bcfc51d219e5fd1559acb0b55408b666768e7d
    generation: 2343
    image: sha256:0195d0f538f0e6b94c723600d9bcfc51d219e5fd1559acb0b55408b666768e7d
tag: installer
```

https://coreos.slack.com/archives/CBN38N3MW/p1627659311031600?thread_ts=1627590256.188300&cid=CBN38N3MW
When @vadim 's repo onboarded, the mapping will be added back.

Reply:
Yup, 4.8 installer and machine-config-daemon are being built locally. I'll onboard my repo next week